test/hooks/quoteTokenPriorities.hooks.test.tsx

```diff
@@ -0,0 +1,147 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { act } from 'react-dom/test-utils'
```

Comment:
Why suddenly do native react renders, when we use `enzyme` in all other component tests?

Reply:
For hook testing enzyme works? I honestly just wrote it from memory using act as I'm used to it. Can change if important